kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/LogSegment.java

```diff
@@ -263,4 +269,8 @@ private String formatList(List<FileStatus> list) {
     return "\n    "
         + list.stream().map(FileStatus::toString).collect(Collectors.joining(",\n    "));
   }
+
+  public Optional<FileStatus> getLatestChecksum() {
```

Comment:
nit: this should be next to the other gettings, and it should be inserted to be in the same order as the parameter in the constructor